Source/Core/Core/PowerPC/Interpreter/Interpreter_LoadStore.cpp

```diff
@@ -997,8 +1001,8 @@ void Interpreter::lwarx(UGeckoInstruction inst)
   if (!(PowerPC::ppcState.Exceptions & EXCEPTION_DSI))
   {
     rGPR[inst.RD] = temp;
-    m_reserve = true;
-    m_reserve_address = address;
+    PowerPC::interpreter.m_reserve = true;
```

Comment:
A better first step would have been to unravel the interpreter's functions and make them non-static before munging around all the table code in general, so this sort of thing isn't necessary from within the interpreter.